VERSIONING.md

```diff
@@ -135,3 +162,10 @@ on its intended usage (e.g. when calling a method or when passed to any method o
     * Contrib modules will be kept up to date with this project's releases.
 * GitHub releases will be made for all releases.
 * Go modules will be made available at Go package mirrors.
+
+### Long term support
```

Comment:
I feel like this needs a "after v1" remark.

Reply:
Clarified, thanks!